index.html

```diff
@@ -5,8 +5,69 @@
     <title>HTML Academy: Глейси</title>
   </head>
   <body>
+    <header>
+      <nav>
+        <img class="logo" src="img/logo.jpg" width="150" height="50" alt="GlacyShop">
+        <ul class="main-navigation">
+          <li><a href="catalog.html">Каталог</a></li>
```

Comment:
```suggestion
          <li>
            <a href="catalog.html">Каталог</a>
          </li>
```

Лучше всегда старайся такую вложенность не делать(точнее отбивать пробелы и переносить на след строку) твои коллеги тебе потом спасибо скажут)